src/models/raffle.js

```diff
@@ -69,7 +69,8 @@ module.exports.pickWinner = function(from, to, localpart) {
     args);
 };
 
-module.exports.listEntries = function(from, to, localpart) {
+module.exports.listEntries = function(from, to, localpart, orderBy, limit) {
```

Comment:
you might consider passing this in as an object, as it stands you have to pass an orderBy to pass a limit. Maybe that's fine for now.